code/traffic-register/traffic-register.js

```diff
@@ -551,3 +557,110 @@ $('#view_657 tbody input[type=checkbox]:checked').each(function() {
   var id = $(this).closest('tr').attr('id'); // record id
 });
 */
+
+/*************************************************************/
+/*** Automated Regulation Text popluted by regulation type ***/
+/*************************************************************/
+
+var automatedRegulationTextPatternsByType = {
```

Comment:
For those of you who have not looked at a lot of Knack code, we lean towards older JS syntax for compatibility purposes. That is why you see a bunch of vars and for loops in here. Also, if that is no longer accurate, please let me know! 🙏